core/controllers/editor_test.py

```diff
@@ -554,8 +554,8 @@ def test_exploration_download_handler_for_default_exploration(self):
 
         # Check download to JSON.
         exploration.update_objective('Test JSON download')
-        exp_services._save_exploration(  # pylint: disable=protected-access
-            owner_id, exploration, '', [])
+        exp_services.update_exploration(
+            owner_id, exploration.id, [], '')
```

Comment:
@aggarwalpulkit596, This won't change the objective of the exploration, right? (You have to add a change command in the change list to update the objective of the exploration.)

Reply:
I didn't understand can you explain a bit more this being the PR I'm not that much familiar with the project

Reply:
I would suggest you read the docstring for the `update_exploration` and see how can you update the exploration objective using this method. (We have to update the objective as per line 556)

Reply:
Yeah got it thanks

Reply:
i understand what do i need to change but i couldn't figure out what cmd do i have to write in changeList

Reply:
Maybe the ExplorationChange class in exp_domain.py will be helpful.

Reply:
yes got it thanks

Reply:
@DubeySandeep can you please check now